app/assets/javascripts/ajax.js

```diff
@@ -0,0 +1,9 @@
+function sendFormSubmissionAjax(dataObj, selector) {
```

Comment:
This is looking good! Perhaps this file needs a more descriptive name, however; but is it a generic utility? If so, maybe can add a sentence or two on how to use it?

Thanks!

Reply:
Yes, I should do that!